locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -6106,34 +6137,34 @@ msgid "Who We Are"
 msgstr "Em Kî Ne"
 
 msgid "Why Privacy"
-msgstr ""
+msgstr "Çima Taybetî"
 
 msgctxt "image-layout"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. http://i.imgur.com/twR2utt.png
 msgctxt "settingsvalue"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. http://i.imgur.com/SOkVdkM.png
 msgctxt "width"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. https://duckduckgo.com/params under "Look & Feel Settings"
 msgid "Width:"
-msgstr ""
+msgstr "Ferehî"
```

Comment:
Mhm, this should be `Width:`, but it looks the same as Wide. Can you please double check and add the `:`, please?